### ISSUE TITLE
Why I have  mbox_loss = 0 (* 1 = 0 loss) or nan?

### ISSUE THREAD

**Issue by nyanmn** [jinfagang/TrafficLightsDetection#7]
I train my own dataset and I always have  mbox_loss = 0 (* 1 = 0 loss) or NAN.
I have all default parameters except my own image has 1280x720 size and I set

resize_width = 512
resize_height = 512
base_lr = 0.0004
batch_size = 16
accum_batch_size = 16
What could be wrong?
